source/module_cell/klist.cpp

```diff
@@ -308,7 +308,12 @@ bool K_Vectors::read_kpoints(const std::string &fn)
 
         ifk >> nmp[0] >> nmp[1] >> nmp[2];
 
```

Comment:
1. you can add initialization of `koffest` before valuing it. 
2. be clear about format